apps/code-examples/src/app/code-examples/popovers/dropdown/basic/dropdown-demo.component.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { expect } from '@skyux-sdk/testing';
+import { SkyDropdownHarness } from '@skyux/popovers/testing';
+
+import { DropdownDemoComponent } from './dropdown-demo.component';
+import { DropdownDemoModule } from './dropdown-demo.module';
+
+fdescribe('Basic dropdown', async () => {
+  async function setupTest(): Promise<{
+    dropdownHarness: SkyDropdownHarness;
+    fixture: ComponentFixture<DropdownDemoComponent>;
+  }> {
+    const fixture = TestBed.createComponent(DropdownDemoComponent);
+    const loader = TestbedHarnessEnvironment.documentRootLoader(fixture);
+    const dropdownHarness = await loader.getHarness(
+      SkyDropdownHarness.with({
+        dataSkyId: 'dropdown-demo',
+      })
+    );
+
+    return { dropdownHarness, fixture };
+  }
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [DropdownDemoModule],
+    });
+  });
+
+  it('should display the correct dropdown', async () => {
+    const { dropdownHarness, fixture } = await setupTest();
+
+    fixture.detectChanges();
+
+    await expectAsync(dropdownHarness.getButtonStyle()).toBeResolvedTo(
+      'default'
+    );
+    await expectAsync(dropdownHarness.getButtonType()).toBeResolvedTo('select');
+    await expectAsync(dropdownHarness.isDisabled()).toBeResolvedTo(false);
+    await expectAsync(dropdownHarness.getAriaLabel()).toBeResolvedTo(
+      'Context menu'
+    );
+    await expectAsync(dropdownHarness.getTooltipTitle()).toBeResolvedTo(null);
+    await expectAsync(dropdownHarness.isOpen()).toBeResolvedTo(false);
+  });
+
+  it('should open the correct dropdown menu', async () => {
+    const { dropdownHarness, fixture } = await setupTest();
+
+    fixture.detectChanges();
+    await dropdownHarness.clickDropdownButton();
+    fixture.detectChanges();
+
+    const dropdownMenu = await dropdownHarness.getDropdownMenu();
+    const dropdownMenuItem = await await dropdownMenu?.getItemAtIndex(0);
+
+    await expectAsync(dropdownHarness.isOpen()).toBeResolvedTo(true);
+    await expectAsync(dropdownMenu?.getRole()).toBeResolvedTo('menu');
+
+    await expectAsync((await dropdownMenuItem?.host())?.text()).toBeResolvedTo(
+      'Option 1'
+    );
```

Comment:
should we showcase this to users on how to get at these values, give them a function that does this, or remove this all together